mappingengine/featureslist/supportedfeatures.rst

```diff
@@ -0,0 +1,82 @@
+===================================
+Supported Mapping Language Features
+===================================
+
+The FHIR Mapper implements a broad set of features of the FHIR Mapping Language so that you use these transformations in a variety of use cases. Please note that the FHIR Mapping Language is still under active development by the FHIR community. The following page provides an overview of all features that are supported by the our engine.
+
+Metadata
+-------------
+Metadata can be added to a StrcutureMap based on a FHIR Mapping Language script by using a /// comment at the beginning of the file. It's possible to set:
+
+- StrcutureMap.title
+- StrcutureMap.version
+- StrcutureMap.status
+
+::
+
+  /// version = 0.1
+  /// title = "FHIR Mapper Tutorial : FakeInpatientDrugChart"
+  /// status = draft
+
+  map "http://vonk.fire.ly/fhir/StructureMap/FHIRMapperTutorial" = FHIRMapperTutorial
+
+
+Flow control
+-------------
+In some cases it is necessary to control which order mapping groups are executed. It's possible to chain mapping groups in order for them to be evaluated subsequently ::
+
+  group MapFakeInpatientDrugChart(source src: FakeInpatientDrugChart, target bundle: Bundle)
+  {
+    src -> bundle.id = uuid();
+    src -> bundle.entry as entry, entry.resource = create('Patient') as patient then
+      TransformPatient(src, patient), TransformPatientPostHandler(src, patient, bundle);
+  }
+
+Here TransformPatient is called first followed by TransformPatientPostHandler, allowing to refine the created Patient resource or to use its content in another resource mapping.
+
+Transformation functions
+------------------------
+The FHIR Mapping language defines a series of transformation functions that can alter the source content before being copied to a target. The following functions are currently supported by the FHIR Mapper.
+
+1. create('<type>') - Explicitly create an element to pass it into a subsequent rule / mapping group::
+  
+  src.guardian as guardian -> patient.contact = create('BackboneElement') as contact collate then {...}
```

Comment:
Maybe don't use `collate` here not to take away from the example. Also `collate` needs to be documented on its own